offline/cable_LUC_EXPT.F90

```diff
@@ -337,7 +337,7 @@ SUBROUTINE LUC_EXPT_INIT(LUC_EXPT)
                      start=(/land_x(k),land_y(k),1/), &
                      count=(/1,1,tdimsize/) )
                 CALL HANDLE_ERR(STATUS, "Reading direct from "//LUC_EXPT%TransFile(i) )
-                !IF (sum(tmpvec).gt.1e-3 .OR. LUC_EXPT%primaryf(k).lt.0.99) LUC_EXPT%prim_only(k) = .FALSE.
+                !IF ((sum(tmpvec) .gt. 1e-3) .OR. (LUC_EXPT%primary_veg(k) .lt. 0.99)) LUC_EXPT%prim_only(k) = .FALSE.
```

Comment:
Delete these two commented out lines entirely?

Reply:
Can delete for the next commit.